common/config/src/lib.rs

```diff
@@ -205,7 +200,7 @@ impl Config {
         Config::try_from(raw_config)
     }
 
-    fn process_logdna_env_vars() {
+    pub fn process_logdna_env_vars() {
```

Comment:
Just curious as to why you chose to make this public and explicitly call in main, as opposed to keeping private and calling within the existing `new()` constructor?

Reply:
it was like that before. it delta diff with per commit. full diff in Files changed.